js/style/style.js

```diff
@@ -323,6 +323,14 @@ Style.prototype = util.inherit(Evented, {
         return this;
     },
 
+    setLayerZoomRange: function(layerId, minzoom, maxzoom) {
+        this.batch(function(batch) {
+            batch.setLayerZoomRange(layerId, minzoom, maxzoom);
+        });
+
+        return this;
+    },
```

Comment:
btw, if you do `return this` inside `batch`, you can cut a few lines of code by doing `return this.batch(...);`